.vscode/settings.json

```diff
@@ -54,4 +54,4 @@
       "url": "./build-system/json-schemas/visual-tests.json"
     }
   ]
-}
+}
```

Comment:
Can you also revert this?

Reply:
Hi @powerivq, we have run the given command. Please check and confirm

Reply:
@mmprebid The file is still here, so it is not right. This file should disappear from this PR.

Reply:
@powerivq 
Hi, Do we need to remove this file completely or remove some code from this file. As this file is also exists in the ampproject and we have forked the repo. 

ampproject/amphtml/.vscode/settings.json

Please let us know.

Reply:
@mmprebid I meant that this file should remain unchanged, and not to delete this file. You are change a newline from this file right now.

Reply:
@powerivq
We have updated the file. please check and let us know

Reply:
@mmprebid It still is not working, as I still see this file on here. Could you try accepting my suggested change below?

Reply:
@powerivq 
We have done syncing with latest updates and we have also compared the files from ampproject/amphtml. Now, Please check and let us know.

If it is not updated then please ignore this file **/.vscode/settings.json** during commit or revert the old code as we are not getting any mismatch on our repository.